src/coreclr/nativeaot/Runtime/amd64/MiscStubs.asm

```diff
@@ -3,6 +3,9 @@
 
 include AsmMacros.inc
 
+EXTERN RhpCidResolve : PROC
```

Comment:
It seems that this change is not needed as none of these two is used in the code here

Reply:
Agreed.